glide-core/redis-rs/redis/src/types.rs

```diff
@@ -823,6 +824,17 @@ impl From<FromUtf8Error> for RedisError {
     }
 }
 
+impl From<ParseIntError> for RedisError {
+    fn from(_: ParseIntError) -> RedisError {
+        RedisError {
+            repr: ErrorRepr::WithDescription(
+                ErrorKind::TypeError,
+                "Cannot parse string as an integer",
+            ),
+        }
+    }
+}
```

Comment:
I decided to add this error to RedisError, despite the fact that this error doesn't originate from Redis/Valkey itself. This is because our FFI code already treats all errors it returns as RedisErrors, regardless of whether Redis responded with that error or we constructed it ourselves in the FFI layer via `RedisError::from`. This might not be desirable, but creating a new error type is a larger change that would warrant a separate PR.